lib/index.d.ts

```diff
@@ -831,7 +831,7 @@ declare namespace Stardog {
     export namespace user {
 
         interface User {
-            username: string;
+            name: string;
```

Comment:
i know this is annoying, but can you revert this change? we have plans to address the inconsistency with this type in `VET-2366`

Reply:
I will make an effort to revert it. I recall this causing me a lot of pain yesterday, and I was shocked that the type had been wrong for such a long time. Lol. But I'll see if there's some workaround I can do to get typescript to look the other way.

So, I presume Stardog.js will eventually require this to be `username` rather than `name`? So I guess in my code, I'll set both `name` and `username` to the same value, so it future-proofs the upcoming fix. And I'll add a comment to fix the hack once VET-2366 is done